version/version.go

```diff
@@ -36,11 +35,6 @@ var (
 func init() {
 	var err error
 	operatorVer := OperatorVersionString
-	// If dockerinfo has a tag (e.g., specified by LDFlags), we will use it as the version of operator
-	tag := buildversion.DockerInfo.Tag
-	if tag != "" && tag != "unknown" {
```

Comment:
instead we can just check version format here for those dev images: 
```
	if tag != "" && buildversion.IsVersionString(tag) {
```

Reply:
we should never get a SHA anyways afaik, how is this built?

Reply:
nevermind, I see how it can happen